### PR TITLE
chore: use improved design of rocket

### PR DESCRIPTION
**Pull request by daKmoR**
What I did (mostly by updating a dependency):
- more white space
- different fonts
- cleaner header (no color background)
- use new rocket cli
- configure rocket via new options in the config file

ToDo:
- [x] alignment of front page image
- [ ] improve dark mode

@jorenbroekema can you finish it up 🙏

also please add before/after screenshots here so we can post them later on twitter 🤗

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -1070,62 +1070,63 @@
     human-id "^1.0.2"
     prettier "^1.18.2"
 
-"@d4kmor/cli@^0.2.5":
-  version "0.2.5"
-  resolved "https://registry.yarnpkg.com/@d4kmor/cli/-/cli-0.2.5.tgz#04d2d6ed0b2baf47ed6d69502f8de172d8e73d83"
-  integrity sha512-uoh5xF5GyYXIwM7LyuvRQQdqas2bTFpEr6fmS9uSaJU7DutiXCxumwSd0uB4cskWGjhdXvpfBvXF2qSBpmeNwQ==
+"@d4kmor/cli@^0.3.2":
+  version "0.3.2"
+  resolved "https://registry.npmjs.org/@d4kmor/cli/-/cli-0.3.2.tgz#5736b6824655bad06a0b5eda214af4d12b1cd400"
```
Comment (LarsDenBakker) [inline yarn.lock:1075]: why did the registry change?

Comment (daKmoR) [reply yarn.lock:1075]: huh... no clue 😅 
maybe I had connection issues and it did fall back to npm registry instead of yarn? 🤷‍♂️

Comment (daKmoR) [reply yarn.lock]: fixed

**Comment by daKmoR** [comment]
## Before

Screenshot 2020-09-14 at 22 11 50

## After

Screenshot 2020-09-14 at 22 12 07

**Pull request merged**